zerver/tests/test_events.py

```diff
@@ -746,7 +746,7 @@ def get_checker(check_gravatar: Validator) -> Validator:
         )
 
         events = self.do_test(
-            lambda: do_update_message(self.user_profile, message, topic,
+            lambda: do_update_message(self.user_profile, message, None, topic,
                                       propagate_mode, content, rendered_content,
```

Comment:
We will want to add a `test_events` test similar to the existing one for changing the stream.  

I opened https://github.com/zulip/zulip/issues/14101 for this; I think it is likely enough of a rabbit hole that we shouldn't go down it before merging this PR.